cypress/component/delegation-accordion.cy.tsx

```diff
@@ -0,0 +1,151 @@
+import * as cypress from '@testing-library/cypress';
+
+import { ApiDelegationAccordion } from '../../src/components/ApiDelegationOverviewPage/ApiDelegationAccordion/ApiDelegationAccordion';
+
+describe('DelegationAccordion Header', () => {
+  it('should show delete button when it contains orgs that are not soft deleted', () => {
```

Comment:
```suggestion
  describe('AccordionHeader', () => {
  it('should show delete button when it contains orgs that are not soft deleted', () => {
```
Add describe to have nice structure when running yarn coverage

Reply:
Ok, now I'm confused...
This is exactly what I've done above, isn't it? 😅

Reply:
Nope, you should test the changes and run yarn coverage